Resources/views/line/video.html.twig

```diff
@@ -0,0 +1,27 @@
+{% set videoType = content.getField( 'file' ).value.mimeType|default( 'video/mp4' ) %}
+
+<div class="content-view-line">
+    <article class="class-video row">
+        <div class="col-md-4">
+            <div class="attribute-video">
+                {{ ez_render_field( content, 'file', {
+                    'attr': { 'id': 'video_' ~ content.id, 'class': 'video-js vjs-default-skin', 'data-setup': '{}' },
+                    'parameters': { 'width': '100%', 'height': '113', 'videoType': videoType },
```

Comment:
@crevillo I agree. It's a lot of work for the browser if the page has 10 videos. But it's not the case for the DemoBundle. There is only one video.